config.py

```diff
@@ -5,7 +5,10 @@ class Config(object):
     SECRET_KEY = os.urandom(64).hex()
 
     NGINX_PATH = '/etc/nginx'
-    CONFIG_PATH = os.path.join(NGINX_PATH, 'conf.d')
+    CONFIG_PATH = os.getenv(
+        'CONFIG_PATH', os.path.join(NGINX_PATH, 'conf.d'))
+
+    print(CONFIG_PATH)
```

Comment:
Probably don't need to print the config path

Reply:
forgot that, thanks